R/MzTabReader.R

```diff
@@ -153,6 +153,11 @@ getEvidence = function()
   
   res = data.table::as.data.table(.self$sections$PSM)
   
+  # remove empty PepIDs (with no ConsensusFeature group or unassigned PepIDs (-1); corresponding to unidentfied MS2 scans)
+  if("opt.global.cf.id" %in% colnames(res)){
```

Comment:
where does this come from? I don't see it anywhere in the current codebase. Should be a remnant of the merge (line 162 has the newer version. So remove 157-159